crates/vault-registry/src/lib.rs

```diff
@@ -131,6 +131,10 @@ pub mod pallet {
             log::info!("Off-chain worker started on block {:?}", n);
             Self::_offchain_worker();
         }
+
+        fn on_runtime_upgrade() -> frame_support::weights::Weight {
```

Comment:
To actually perform the migration, you need to bump the spec versions. I'm not sure what other changes this will be batched with, so I'm not sure if that should be done here or in a separate pr. @gregdhill ?

Reply:
Let's do that in a separate PR :)